narayana-release-process.sh

```diff
@@ -43,10 +35,6 @@ then
   exit
 fi
 
-# Do this early to prevent later interactive need
-docker login docker.io
```

Comment:
Until we have the next minor we would probably push updates to docker but as we have a 5.10 so we don't need to do this for this branch still but in future we would think when is the right time to remove this step

Reply:
same observation would apply to the WFLYISSUE I suppose